e2e/app/page/notebook-preview-page.ts

```diff
@@ -34,6 +36,9 @@ export default class NotebookPreviewPage extends AuthenticatedPage {
     const editLink = await this.getEditLink();
     await editLink.waitUntilEnabled();
     await editLink.click();
+
+    await initializeRuntimeIfModalPresented(this.page);
```

Comment:
@aweng98 FYI , new interaction here . I believe I've covered the touchpoints in puppeteer.